src/View/SSViewer.php

```diff
@@ -557,7 +586,9 @@ public function process($item, $arguments = null, $inheritedScope = null)
         if ($this->rewriteHashlinks && $rewrite) {
             if (strpos($output, '<base') !== false) {
                 if ($rewrite === 'php') {
-                    $thisURLRelativeToBase = "<?php echo \\SilverStripe\\Core\\Convert::raw2att(preg_replace(\"/^(\\\\/)+/\", \"/\", \$_SERVER['REQUEST_URI'])); ?>";
+                    $thisURLRelativeToBase = <<<PHP
+<?php echo \\SilverStripe\\Core\\Convert::raw2att(preg_replace("/^(\\\\/)+/", "/", \$_SERVER['REQUEST_URI'])); ?>
+PHP;
```

Comment:
Change is mainly because of line length, but a happy side-effect is that this negates the need for escaping quotes in the already complex regex